### ISSUE TITLE
Ads serving has been limited

### ISSUE THREAD

**Issue by MaddPenguin** [kmcgill88/admob_flutter#149]
Pardon me if this is an unrelated issue. I am new to flutter development and in serious need of help.

This is my second time receiving such message. 

I am unsure if my implementation is wrong or something else.

My app:
- Admob initializes in my main.dart
- on second page i have a list view with multiple ads (max 2) in the list. There's multiple drop-down button which triggers setstate as user make a selection.

Are the multiple setStates the cause to my issue.

Flutter : v1.12.13+hotfix.8
admob_flutter: "^0.3.4"

-------------------------------------
I've followed the guide as stated in [this](https://medium.com/@vitopiegari/embedding-ads-into-flutters-widget-tree-with-admob-flutter-ae59c3a66492) Medium article. But still getting this issue.

Anyone has a solution or a reference for me is much appreciated.

**Comment by MarcoSoares11** [comment]
I went through the same problem a few weeks ago... my solution for this was to make another Admob App (of the same app I'm developing), and connecting it to the Google Play Store by linking. This solved the problem for me, and from what I know, it is caused for a few reasons, including clicking on your own ads, that is against the Admob policies.

**Comment by anilbhomi** [comment]
I am having the same issue. I don't know where I went wrong
@MarcoSoares11 what do you mean by "make another AdMob app"?
Does it mean to purchase a new developer account and upload the app with the new package name?

**Comment by MarcoSoares11** [comment]
@anilbhomi No, what I mean is simply go to your Admob console and on (Apps > Add app) and create a new admob app for your same app. Now just transfer over all the Ad Units and just change the Admob App ID on your app config, (AndroidManifest and wherever you have defined the value of Admob App ID)... this worked for me, and now my ads are displaying normally.

**Comment by anilbhomi** [comment]
I have already linked my app to Admob and now I can't undo it according to documentation. Do you have any idea regarding this because my app still has that particular issue? Even if I do as per your suggestion I need to publish the same app with the different package name.

**Comment by Mouadabdelghafouraitali** [comment]
Does anyone have a solution to this issue.

**Comment by kellvembarbosa** [comment]
Do your application have an "iframe" or webview?
Usually when I receive this warning it is via webview (iframe) in the app ...

To solve the problem I launch an update and mark the problem as solved ...
90% of the time it works, and only when you have a webview and it doesn't work!

To mark it as resolved go to the policy center and click resolved, remember to send a new apk before ...
